samples/bootloader/prj.conf

```diff
@@ -30,3 +30,10 @@ CONFIG_ASSERT=n
 
 # Avoid triggering IRQs from the RTC
 CONFIG_NRF_RTC_TIMER=n
+
+CONFIG_CLOCK_CONTROL_NRF=n
+CONFIG_NOTIFY=n
+CONFIG_ONOFF=n
+
+CONFIG_BOOT_BANNER=n
+CONFIG_NCS_BOOT_BANNER=n
```

Comment:
should not be applying this for everything, add a `boards/<board>` Kconfig fragment, and it should have a file suffix because people expect the default one to all act the same way which is with logging